packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -94,11 +108,22 @@
 }
 
 .description {
-  color: var(--calcite-tile-description-text-color);
+  color: var(--calcite-tile-description-text-color, var(--calcite-internal-tile-description-text-color));
   font-weight: var(--calcite-font-weight-regular);
   overflow-wrap: break-word;
 }
 
+calcite-icon {
+  --calcite-icon-color: var(
+    --calcite-tile-icon-color,
+    var(--calcite-internal-tile-icon-color, var(--calcite-color-text-3))
+  );
+
+  &.selection-icon {
+    --calcite-icon-color: var(--calcite-internal-tile-select-icon-color);
```

Comment:
You'll need to set the `color` here for the variable to take effect, e.g. `color: var(--calcite-icon-color);`.  I'm also wondering if `--calcite-icon-color` should be renamed to map to the internal counterpart, e.g. `--calcite-tile-icon-color`?